### PR TITLE
Better help on parameterized breeze commands

### PR DESCRIPTION
**Pull request by potiuk**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: breeze [hunk 1]
```diff
@@ -535,7 +535,7 @@ function prepare_command_files() {
 
     # Prepare script for "run test"
     prepare_command_file "${BUILD_CACHE_DIR}/${LAST_DC_TEST_CI_FILE}" \
-        "${DC_RUN_CI_COMMAND}" "${COMPOSE_CI_FILE}" "true" "${AIRFLOW_CI_IMAGE}" '*'
+        "${DC_RUN_CI_COMMAND}" "true" "${COMPOSE_CI_FILE}" "${AIRFLOW_CI_IMAGE}" '*'
```
Comment (zhongjiajie) [inline breeze]: Why I merged the base PR but it still here

File: breeze [hunk 2]
```diff
@@ -804,14 +804,20 @@ function parse_arguments() {
...
+          if [[ $# -lt 2 ]]; then
             echo "You should specify docker compose command to run"
-            exit 1
+            shift
+            RUN_HELP="true"
+          else
+            DOCKER_COMPOSE_COMMAND="${2}"
+            if [[ ${DOCKER_COMPOSE_COMMAND} == "--help" ]]; then
+                RUN_HELP="true"
+            fi
```
Comment (zhongjiajie) [inline breeze]: I test in local and seem delete this three code and `--help` option still work, also the short option `-h`

File: breeze (hunk 1 above)
Comment (potiuk) [reply breeze]: I need to rebase :)

File: breeze (hunk 2 above)
Comment (potiuk) [reply breeze]: Right. I forgot that the --options are parsed before and removed :). Good catch. Fixed.

Comment (potiuk) [reply breeze]: I actually found that there was also one thing missing (specification of the command parameter for parameterized command). I added it.

File: BREEZE.rst [hunk 3]
```diff
@@ -1046,15 +1046,21 @@ This is the current syntax for  `./breeze <./breeze>`_:
...
+                 all all-but-pylint airflow-config-yaml base-operator bat-tests build
+                 build-providers-dependencies check-apache-license check-executables-have-shebangs
+                 check-hooks-apply check-integrations check-merge-conflict check-xml
+                 consistent-pylint daysago-import-check debug-statements detect-private-key doctoc
+                 end-of-file-fixer fix-encoding-pragma flake8 forbid-tabs
+                 incorrect-use-of-LoggingMixin insert-license isort lint-dockerfile mixed-line-ending
+                 mypy provide-create-sessions pydevd pylint pylint-tests python-no-log-warn
+                 rst-backticks setup-order shellcheck stylelint trailing-whitespace
+                 update-breeze-file update-extras update-local-yml-file update-setup-cfg-file
+                 yamllint
```
Comment (turbaszek) [inline BREEZE.rst:1063]: Does it work with autocomplete?

Comment (potiuk) [reply BREEZE.rst:1063]: Yes. that's the whole point of having ti as pre-commit does not have auto-complete.

File: breeze (hunk 1 above)
Comment (zhongjiajie) [reply breeze]: Got thanks

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**